notebooks/Setting up a labeling task.ipynb

```diff
@@ -128,18 +128,14 @@
     "\n",
```

Comment:
<p>Can we now use the csv here too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>eliminate all dependency on firestore and buckets </p>